### DISCUSSION TITLE
About probability_modified threshold

### DISCUSSION THREAD

**Discussion by Yang990-sys** [GoekeLab/m6anet#114] [Q&A] [answered]
Hello，
I counted the correlation between prob and mod in the example results and found that they showed a positive correlation. When prob>0.9, the values of mod are all>0.5. Does this prove that m6anet's prediction of low-level methylation sites is not so accurate? Or is the calculation process of Prob related to mod? How to explain this strong correlation?  
![image](https://github.com/GoekeLab/m6anet/assets/54176741/97b3984d-dcd7-4108-bd1b-23a9a93b1001)

Thanks very much!

**Comment by chrishendra93** [comment, marked as answer]
hi @Yang990-sys , the mod ratio will definitely be correlated with probability modified. The training data of m6Anet comprises modification labels at the site levels and so we need some way to pool the probability output of m6anet for each read. 

Probability modified essentially serves this purpose - we reason that the m6ACE labels in our training data will pick up sites with at least one modified read, and the higher the number of modified reads, the higher the probability that this site will be picked up by m6ACE. So high probability modified indicates that this site is likely to be picked up as modified if we are to run m6ACE on the dataset. 

The mod_ratio, on the other hand, is not trained directly on m6ACE labels. The probability modified is by no means quantitative, but since we output modification probability for each read, we can then set a threshold that will allow us to say whether a read is modified or not. Using this threshold, we can calculate the percentage of modified reads per site in the form of mod_ratio which is also more interpretable than probability_modified. High mod_ratio number suggests that a lot of reads are modified in a sense that a lot of them have read level modification probability above a certain threshold - which will translate to high probability_modified by design

**Reply by Yang990-sys** [reply]
Thank you!
